### DISCUSSION TITLE
Increasing font size in plotEnrichHeatmap

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#996] [Questions / Documentation] [answered]
I have been trying to increase the fontsize to no avail.

Could you some help.

Thanks!

**Comment by rcorces** [comment, marked as answer]
Not everything is customizable in ArchR. You are welcome to extract the corresponding matrix using `returnMatrix` and plot it however you want.

**Reply by sylestiel** [reply]
Thank You!
